app/controllers/performance_reports_controller.rb

```diff
@@ -0,0 +1,67 @@
+class PerformanceReportsController < ApplicationController
+  include MakeNotice
```

Comment:
Maybe we can move this into `ApplicationController`? It's already used in most controllers anyway.

Reply:
true